.github/workflows/paladin-PR-build.yml

```diff
@@ -49,6 +49,15 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew -PcomposeLogs=true -PverboseTests=true --no-daemon --parallel --max-workers=5 build
   
+
+      - name: Upload logs
+        uses: actions/upload-artifact@v4
+        if: always()      
```

Comment:
is this needed?

Reply:
Yeah - I believe so, otherwise on failure (when we need the logs most), the step is skipped